src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -132,25 +130,55 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             },
             handleError: "{that}.events.onError.fire",
             handlePause: {
-                changePath: "paused",
-                value: true
+                funcName: "fluid.textToSpeech.handlePause",
+                args: ["{that}"]
             },
             handleResume: {
-                changePath: "paused",
-                value: false
+                funcName: "fluid.textToSpeech.handleResume",
+                args: ["{that}"]
             }
         }
     });
 
+    // Issue commands to the speechSynthesis interface after a delay; this
+    // makes the wrapper behave somewhat better when issuing commands, especially
+    // play and pause
+    fluid.textToSpeech.asyncSpeechSynthesisControl = function (control, delay, args) {
+        setTimeout(function () {
+            speechSynthesis[control](args);
+        }, delay);
+    };
+
     fluid.textToSpeech.speak = function (that, speaking) {
         that.events[speaking ? "onStart" : "onStop"].fire();
     };
 
-    fluid.textToSpeech.pause = function (that, paused) {
-        if (paused) {
-            that.events.onPause.fire();
-        } else if (that.model.speaking) {
-            that.events.onResume.fire();
+    fluid.textToSpeech.handlePause = function (that) {
+        that.applier.change("paused", true);
+        that.events.onPause.fire();
+        // Clear to issue any waiting resume command
+        fluid.textToSpeech.clearControlRequest(that, "resumeRequested", "resume");
+    };
+
+    fluid.textToSpeech.handleResume = function (that) {
+        that.applier.change("paused", false);
+        that.events.onResume.fire();
+        // Clear to issue any waiting pause command
+        fluid.textToSpeech.clearControlRequest(that, "pauseRequested", "pause");
+    };
+
+    fluid.textToSpeech.clearControlRequest = function (that, modelBoolPath, controlName) {
```

Comment:
Does it make sense per comment above about Rule of Thumb to simply move `fluid.debounce` from the jqUnit implementation and borrow the Underscore implementation of throttle for `fluid.throttle`? Since I have need of them here, I can do this as part of the PR.